kit/ChildSession.cpp

```diff
@@ -2471,6 +2471,7 @@ void ChildSession::loKitCallback(const int type, const std::string& payload)
         break;
     case LOK_CALLBACK_CELL_CURSOR:
         sendTextFrame("cellcursor: " + payload);
+        sendTextFrame("textselection:");
```

Comment:
Not a good idea to create fake events. It makes maintenance a nightmare. 

Better to find the root cause and fix it, instead of hacking messages. I'm fairly certain this is a regression as it worked in the past. Bisect is your friend.

Reply:
I'll try to find the root cause, but this is not regression it has been there for almost 4 years @thebearon suggested.

Reply:
@lpranam 
Since the selection stuff is related to JS side, the fix should be in the same place in my opinion. at the end of onCellCursorMsg function you can do something like:
		this._textSelectionStart = null;
		this._textSelectionEnd = null;
		this._selectedTextContent = '';
		for (var key in this._selectionHandles) {
			this._map.removeLayer(this._selectionHandles[key]);
			this._selectionHandles[key].isDragged = false;
		} //to remove markers
		this._selections.clearLayers(); // this will remove the blue bg which we do the same when we select a new text.
This can be made as a separate function and be used in other places as well.

Reply:
@merttumer that was another solution I tried but unfortunately that solution works partially only. When you click on another cell selection and selection handlers disappear. But when you perform some other action like you drag to autofill or perform an undo selection handlers will reappear because we receive `textselection:<some values>`. or by performing any operation which sends that msg

Reply:
@merttumer it was my mistake, I had tried all these things but in different functions and missed `onCellCursorMsg`. But it works fine there :)